Fix building with glibc-2.16

https://bugs.gentoo.org/show_bug.cgi?id=430252

Patch written by Kacper Kowalik <xarthisius@gentoo.org>
--- a/src/common/jobacct_common.h
+++ b/src/common/jobacct_common.h
@@ -55,6 +55,7 @@
 
 #include <dirent.h>
 #include <sys/stat.h>
+#include <sys/resource.h>
 
 #include "slurm/slurmdb.h"
 
--- a/src/salloc/salloc.c
+++ b/src/salloc/salloc.c
@@ -49,6 +49,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/param.h>
+#include <sys/resource.h>
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <termios.h>
--- a/src/sbatch/sbatch.c
+++ b/src/sbatch/sbatch.c
@@ -48,6 +48,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/param.h>               /* MAXPATHLEN */
+#include <sys/resource.h>
 #include <fcntl.h>
 
 #include "slurm/slurm.h"
